scheduler/src/main/java/gov/nasa/jpl/aerie/scheduler/SimulationDriverFacade.java

```diff
@@ -0,0 +1,42 @@
+package gov.nasa.jpl.aerie.scheduler;
+
+import gov.nasa.jpl.aerie.merlin.driver.ActivityInstanceId;
+import gov.nasa.jpl.aerie.merlin.driver.SerializedActivity;
+import gov.nasa.jpl.aerie.merlin.driver.SimulationResults;
+import gov.nasa.jpl.aerie.merlin.protocol.types.Duration;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class SimulationDriverFacade {
+
+  private final IncrementalSimulationDriver driver;
+
+  private final Map<String, ActivityInstanceId> activityNameToActivityId = new HashMap<>();
+
+  private long count = 0;
```

Comment:
Maybe this should be `static`, otherwise `simulateActivity()` may produce falsely-identified `ActivityInstanceId`s if `SimulationDriverFacade` is instantiated and used more than once.

Reply:
Will do.